client/lxd_cluster.go

```diff
@@ -149,3 +149,21 @@ func (r *ProtocolLXD) RenameClusterMember(name string, member api.ClusterMemberP
 
 	return nil
 }
+
+// AddClusterMember generates a join token to add a cluster member.
+func (r *ProtocolLXD) AddClusterMember(name string) (Operation, error) {
```

Comment:
Same as previous, this should be `func (r* ProtocolLXD) CreateClusterMember(member api.ClusterMembersPost) (Operation, error)`